unittests/CppInterOp/FunctionReflectionTest.cpp

```diff
@@ -829,6 +838,19 @@ TEST(FunctionReflectionTest, GetFunctionArgName) {
   EXPECT_EQ(Cpp::GetFunctionArgName(Decls[1], 1), "d");
   EXPECT_EQ(Cpp::GetFunctionArgName(Decls[1], 2), "l");
   EXPECT_EQ(Cpp::GetFunctionArgName(Decls[1], 3), "ch");
+
+  // EXPECT_EQ(Cpp::GetFunctionArgName(Decls[2], 0), "");
```

Comment:
Can you remove the commented code or I comment it.

Reply:
these tests pass, have uncommented